app/src/main/java/info/nightscout/android/model/medtronicNg/StatusEvent.java

```diff
@@ -0,0 +1,301 @@
+package info.nightscout.android.model.medtronicNg;
```

Comment:
Rationale for this is separation of concerns (database vs http API) and specifically being able to write unit tests. But I agree the way the decoupling is being done here is not the greatest. I will look at it again and see if I can find another way.